src/slack/warframe.js

```diff
@@ -112,6 +115,11 @@ function help(requestBody) {
       fallback: "cetus",
       text: `Returns the current time on and bounties cetus e.g. \`${command} cetus\``
     },
+    {
+      title: "alerts-watch",
+      fallback: "alerts-watch",
+      text: `Manages your live alert notifications e.g. \`${command} alerts-watch:[start/stop/add/remove]:[reward]\``
```

Comment:
Provide more detailed examples, similar to your [second comment](https://github.com/MarkNjunge/helix/pull/1#issuecomment-415349830) in this PR.